jetty-core/jetty-http/src/test/java/org/eclipse/jetty/http/MimeTypesTest.java

```diff
@@ -30,6 +30,8 @@ public static Stream<Arguments> mimeTypesByExtensionCases()
     {
         return Stream.of(
             Arguments.of("test.gz", "application/gzip"),
+            Arguments.of("test.tar.gz", "application/gzip"),
+            Arguments.of("test.tgz", "application/x-gtar"),
```

Comment:
This is deliberate, yes? You removed type `application/x-gtar` from mime.properties ...

Reply:
Correct.  Aligning with common extension handling practice of just looking at the final dot segment.  This is more efficient, same as most other projects and there is only one extension we had with a dot in it.